libs/application/templates/parental-leave/src/fields/Review/ReadOnlyReview.tsx

```diff
@@ -0,0 +1,337 @@
+            </Box>
+          </AccordionItem>
+          <AccordionItem
+            id="id_1"
+            label={formatMessage(mm.employer.subSection)}
+            startExpanded={allItemsExpanded}
+          >
+            <Box paddingY={4}>
+              <GridRow>
+                <GridColumn span="12/12">
+                  <Text>
+                    {
+                      getValueViaPath(
+                        application.answers,
+                        'employer.email',
+                      ) as string[]
+                    }
+                  </Text>
+                </GridColumn>
+              </GridRow>
+            </Box>
+          </AccordionItem>
+
+          <AccordionItem
+            id="id_4"
+            label={formatMessage(m.yourRights)}
+            startExpanded={allItemsExpanded}
+          >
+            <Box paddingY={4}>
+              <GridRow>
+                <GridColumn span="12/12">
+                  <YourRightsBoxChart application={application} />
+                </GridColumn>
+              </GridRow>
+            </Box>
+          </AccordionItem>
+
+          <AccordionItem
+            id="id_4"
+            label={formatMessage(m.periodsSection)}
+            startExpanded={allItemsExpanded}
+          >
+            <Box paddingY={4}>
+              <GridRow>
+                <GridColumn span="12/12">
+                  <Timeline
+                    initDate={dobDate}
+                    title={formatMessage(m.expectedDateOfBirthTitle)}
+                    titleSmall={formatMessage(m.dateOfBirthTitle)}
+                    periods={formatPeriods(
+                      application.answers.periods as Period[],
+                      otherParentPeriods,
+                    )}
+                  />
+                </GridColumn>
+              </GridRow>
+            </Box>
+          </AccordionItem>
+
+          <AccordionItem
+            id="id_4"
+            label={formatMessage(mm.paymentPlan.subSection)}
+            startExpanded={allItemsExpanded}
+          >
+            <Box paddingY={4}>
+              <GridRow>
+                <GridColumn span="12/12">
+                  {!loading && !error && (
+                    <PaymentsTable
+                      application={application}
+                      payments={data.getEstimatedPayments}
+                    />
+                  )}
+                </GridColumn>
+              </GridRow>
+            </Box>
+          </AccordionItem>
+
+          <AccordionItem
+            id="id_4"
+            label={formatMessage(mm.shareInformation.subSection)}
+            startExpanded={allItemsExpanded}
+          >
+            <Box paddingY={4}>
+              <GridRow>
+                <GridColumn span="12/12">
+                  <Box marginTop={1} marginBottom={2}>
+                    <Text variant="h5">
+                      {formatMessage(mm.shareInformation.title)}
+                    </Text>
+                  </Box>
```

Comment:
If you're looking for less markup then you could use the margin props on the `Text` component
```suggestion
                    <Text variant="h5" marginTop={1} marginBottom={2}>
                      {formatMessage(mm.shareInformation.title)}
                    </Text>
```

Reply:
Agree, should be cleaned up in the latest as well.